src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs

```diff
@@ -66,8 +66,8 @@ at com.android.dx.command.dexer.Main.main(Main.java:215)
 			at com.android.dx.command.Main.main(Main.java:106)
 		*/
 		const string ExceptionRegExString = @"(?<exception>java.lang.+):(?<error>.+)";
-		Regex codeErrorRegEx = new Regex (CodeErrorRegExString, RegexOptions.Compiled);
-		Regex exceptionRegEx = new Regex (ExceptionRegExString, RegexOptions.Compiled);
+		protected Regex codeErrorRegEx = new Regex (CodeErrorRegExString, RegexOptions.Compiled);
```

Comment:
This should also be PascalCased as `CodeErrorRegex`, as it's not a "publicly visible" member.